MetaMask/metamask-extension
# feat: Network buttons during advanced onboarding

## **Description**

New UI for customizing network RPC during advanced onboarding.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/25276?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. build with `yarn; ENABLE_NETWORK_UI_REDESIGN=1 yarn start`
2. during onboarding, click advanced configuration
3.  Should see the new network buttons, clicks not enabled yet

## **Screenshots/Recordings**



### **Before**

<img width="468" alt="Screenshot 2024-06-12 at 2 28 25 PM" src="https://github.com/MetaMask/metamask-extension/assets/3500406/b81fd542-c504-4b94-a2eb-bd4cc3011050">

### **After**

<img width="478" alt="Screenshot 2024-06-12 at 2 19 58 PM" src="https://github.com/MetaMask/metamask-extension/assets/3500406/0d4f89cc-7702-425b-854c-f8c6356f0c66">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
